manifests/g/Gitlab/gitlab-runner/13.2.0/Gitlab.gitlab-runner.yaml

```
@@ -0,0 +1,20 @@
+PackageIdentifier: Gitlab.gitlab-runner
+PackageVersion: 13.2.0
+PackageLocale: en-US
+Publisher: Gitlab
+PackageName: gitlab-runner
+License: MIT
+LicenseUrl: https://gitlab.com/gitlab-org/gitlab-runner/-/blob/main/LICENSE
+PackageUrl: https://gitlab.com/gitlab-org/gitlab-runner
+Moniker: gitlab-runner
+ShortDescription: GitLab Runner is the open source project that is used to run your CI/CD jobs and send the results back to GitLab
+Installers:
+  - Architecture: x86
+    InstallerType: exe
```

```suggestion
    InstallerType: portable
```
Since this is just a binary, it should be installer type 'Portable'